cmd/libs/go2idl/client-gen/generators/client_generator.go

```diff
@@ -217,6 +217,57 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 	}
 	includedTypesOverrides := customArgs.IncludedTypesOverrides
 
+	// If there is a comment of the form "// +groupName=somegroup" or
```

Comment:
Can you encapsulate the added code in a function?

Is it possible to move the entire parsing to main.go?